src/components/CampaignCard.vue

```diff
@@ -2,7 +2,7 @@
   <div>
     <section class="campaign-card">
       <h1> {{campaign.title}}</h1>
-      <router-link to="/campaigns/0948dx83ds">
+      <router-link :to="{ name: 'Single Campaign', params: {address: campaign.address} }">
```

Comment:
Create new link on card and use campaign address as dynamic route path

Reply:
Nice I like this